### PR TITLE
#147181005 Create Blog on healthchecks

### PR DESCRIPTION
**Pull request by akash-011**
#### What does this PR do?
Create a blog on the healthchecks website
#### Description of Task to be completed?
Blog posts can be viewed on the healthchecks website. Blog posts can be added by users who are logged in. Blogs can be viewed by anybody 
#### How should this be manually tested?
App is live at [Heroku](https://healthblog.herokuapp.com/blog/ )
#### What are the relevant pivotal tracker stories?
Story ID : #147181005

### CODE REVIEW

**Comment by NdagiStanley** [comment]
@akash-011 let's fix the merge conflicts.

**Comment by NdagiStanley** [comment]
Way to go @akash-011

File: hc/settings.py [hunk 1]
```diff
@@ -111,7 +112,7 @@
         }
     }
 
-# Update database configuration with $DATABASE_URL.
+
```
Comment (NdagiStanley) [inline hc/settings.py]: The comment is necessary, in my view. Maybe it just needed editing but not removing it.

File: requirements.txt [hunk 2]
```diff
@@ -8,5 +8,6 @@ futures==3.0.3
 gunicorn==19.7.1
 premailer==2.9.6
 psycopg2==2.6.1
+rcssmin==1.0.6
```
Comment (NdagiStanley) [inline requirements.txt]: This package is included but not used. Please confirm this.

**Review by NdagiStanley — CHANGES_REQUESTED** [review]
I'll have a look at the deployment and maybe we can have a one-on-one if the need arises.

File: requirements.txt (hunk 2 above)
Comment (akash-011) [reply requirements.txt]: Yes , I can confirm this

File: hc/settings.py [hunk 3]
```diff
@@ -17,7 +17,8 @@
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
 HOST = "localhost"
-SECRET_KEY = os.getenv('SECRET_KEY')
+# SECRET_KEY = os.getenv('SECRET_KEY')
+SECRET_KEY = "----"
```
Comment (NdagiStanley) [inline hc/settings.py]: @akash-011 we need to revert this.

**Comment by NdagiStanley** [comment]
@akash-011 You'll need to `git pull --rebase origin develop`

File: hc/settings.py (hunk 3 above)
Comment (NdagiStanley) [reply hc/settings.py]: Please check this.

File: Procfile [hunk 4]
```diff
@@ -1 +1,2 @@
-web: python manage.py makemigrations && python manage.py migrate && gunicorn hc.wsgi:application --log-file -
+web: gunicorn hc.wsgi --log-file -
```
Comment (NdagiStanley) [inline Procfile]: Please revert this @akash-011 .

**Review by NdagiStanley — APPROVED** [review]

**Pull request merged**